api/main_endpoints/util/discord-connection.js

```diff
@@ -0,0 +1,39 @@
+const User = require('../models/User');
+const axios = require('axios');
+const {
+  discordApiKeys
+} = require('../../config/config.json');
+
+function loginWithDiscord(code, email) {
+  return axios
+    .post('https://discord.com/api/oauth2/token',
```

Comment:
`axios.post` supports key value pairs. you can do
```js
.post('https://discord.com/api/oauth2/token',
  client_id: discordApiKeys.CLIENT_ID,
  client_secret: discordApiKeys.CLIENT_SECRET,
  grant_type: 'authorization_code',
  code,
  redirect_uri: discordApiKeys.REDIRECT_URI
})
```

Reply:
errors near the colons as shown below
![image](https://user-images.githubusercontent.com/66745211/89096286-5f894280-d38a-11ea-9cbf-728987e4f6b2.png)

Reply:
add an opening bracket on line 9. ask any more questions in discord please

Reply:
edit: tried this approach and the code will not work at all when written this way. has to be written as it was before